internal/ngap/handler.go

```diff
@@ -912,48 +912,67 @@ func HandleInitialUEMessage(ran *context.AmfRan, message *ngapType.NGAPPDU) {
 	}
 
 	ranUe := ran.RanUeFindByRanUeNgapID(rANUENGAPID.Value)
-	if ranUe != nil && ranUe.AmfUe == nil {
+	if ranUe != nil {
```

Comment:
For this case, amf should send ErrorIndication with CauseRadioNetworkPresentInconsistentRemoteUENGAPID and return. Removing existed ranUe and amfUe is dangerous if any other nas communication is on-going.

Reply:
Our policy is basically to accept subsequent NGAP messages first. Otherwise, it would be difficult to manage UE resources.
Since the RANUENGAPID is generated by the gNB, it would not be appropriate to send an ErrorIndication as if the ID is incorrect.
Also, in this case, it is not dangerous to remove the existing ranUe and amfUe because they will not be used in other nas communication.(since a ranUe is associated with one gNB (AmfRan) and an amfUe is also associated with one ranUe)

Reply:
InitialUEMessage is the NGAP message, so AMF should detach old RanUe and attach new RanUe, and no need to release AmfUe and SMContexts.

Reply:
OK. I withdraw the PR.